.gitignore

```diff
@@ -1,3 +1,4 @@
 .idea
 bin
 vendor
+composer.lock
```

Comment:
My fault, configured the web server and executed chmod through my link. I can switch that back.